src/config/GraphQLConfigTypes.js

```diff
@@ -0,0 +1,36 @@
+/**
+ *  Copyright (c) Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ *  @flow
+ */
+
+export type GraphQLConfiguration = {
+  schemaPath?: string,  // may be .json or .graphql
+  schemaUrl?: string,
+  // If env is specified, use one of the app configs in here
+  env?: GraphQLConfigurationEnvs,
```

Comment:
This also describes the configuration protocol, so I think it'd be good to have it even without the implementation